dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -36,6 +41,19 @@ LABEL \
   org.label-schema.url="{{ .BeatURL }}" \
   org.label-schema.vcs-url="{{ $repoInfo.RootImportPath }}" \
   org.label-schema.vcs-ref="{{ commit }}" \
+  io.k8s.description="{{ .BeatDescription }}" \
+  io.k8s.display-name="{{ .BeatName | title }} image" \
+  org.opencontainers.image.created="{{ date }}" \
+  org.opencontainers.image.licenses="{{ .License }}" \
+  org.opencontainers.image.title="{{ .BeatName | title }}" \
+  org.opencontainers.image.vendor="{{ .BeatVendor }}" \
+  name="{{ .BeatName }}" \
+  maintainer="infra@elastic.co" \
+  vendor="{{ .BeatVendor }}" \
+  version="{{ beat_version }}" \
+  release="1" \
+  url="{{ .BeatURL }}" \
+  summary="{{ .BeatName }}" \
```

Comment:
The [Elasticsearch](https://github.com/elastic/elasticsearch/pull/60742/files) and [Kibana](https://github.com/elastic/kibana/pull/74656/files) images only add these labels to the UBI images. Should we do the same?

Reply:
I don't have an strong opinion, but I guess it doesn't do any harm to have these labels in all images?

Reply:
Yeah I don't think it really matters. Feel free to ignore, just thought I'd mention in case you wanted to make it consistent.